### PR TITLE
Replace tf prefix with tony (make it generic) and update READ.me

### PR DESCRIPTION
**Pull request by oliverhu**
Replace tf prefix with tony (make it generic) and update READ.me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +131,10 @@ Here is a full example of configuring your TonY application:
 
 CLI configurations have highest priority, so we will get 2 ps instances and 2 worker instances. Then the XML file takes next priority so each worker will get 4g memory and 1 GPU. Finally every other configuration will be default value, e.g. each ps will get 2g memory.
 
+#### TonY Examples
+1. [Distributed mnist with TensorFlow](https://github.com/linkedin/TonY/tree/master/tony-examples/mnist-tensorflow)
+2. [Distributed mnist with PyTorch](https://github.com/linkedin/TonY/tree/master/tony-examples/mnist-pytorch)
```
Comment (erwa) [inline README.md]: Nit: change mnist to MNIST here and in the line above

**Comment by erwa** [comment]
Looks like there were some CI test failures related to this change. Can you take a look? Example failure:
```
    2018-10-11 01:33:12 FATAL Utils:114 - Failed to unzip tony.zip
    net.lingala.zip4j.exception.ZipException: zip file does not exist
```

**Comment by oliverhu** [comment]
Fixed unit test, close #27

File: tony-core/src/test/java/com/linkedin/tony/TestTonyClient.java [hunk 2]
```diff
@@ -29,7 +29,7 @@ public void setup() {
 
   @Test
   public void testCreateAMContainerSpec() throws Exception {
-    File zipFile = new File(System.getProperty("user.dir") + "/tf_archive.zip");
+    File zipFile = new File(System.getProperty("user.dir") + "/tony_archive.zip");
```
Comment (erwa) [inline tony-core/src/test/java/com/linkedin/tony/TestTonyClient.java]: Can you reference Constants.TONY_ZIP_NAME here instead?

**Pull request merged**